README.md

```diff
@@ -11,7 +11,7 @@ ECMAScript Stage-0 Proposal. J. S. Choi, 2021.
 
 **Method binding** `->` is a **left-associative infix operator**.
 Its right-hand side is an **identifier** (like `f`)
-or a parenthesized **expression** (like `hof()`),
+or a parenthesized **expression** (like `(hof())`),
```

Comment:
this suggests i can't do `a->f()`, or are you saying that the `a->f` gets resolved before the function call?

Reply:
Yes, `a->f()` is equivalent to `(a->f)()`, not `a->(f())`.

`(a->f)()` is equivalent to `f.bind(a)()`, which is equivalent to `f.call(a)`.